rotkehlchen/tests/api/test_liquity.py

```diff
@@ -248,3 +257,40 @@ def test_account_with_proxy(rotkehlchen_api_server, inquirer):  # pylint: disabl
     # test that the list of addresses was not mutated
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
     assert len(rotki.chain_manager.accounts.eth) == 3
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[JUSTIN, LIQUITY_POOL_DEPOSITOR]])
+@pytest.mark.parametrize('ethereum_modules', [['liquity']])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+def test_stability_pool(rotkehlchen_api_server):
+    """Test that we can get the status of the deposits in the stability pool"""
+    def mock_etherscan_transaction_response(etherscan: 'Etherscan'):
+        def mocked_request_dict(url, *_args, **_kwargs):
+            if '0xeefBa1e63905eF1D7ACbA5a8513c70307C1cE441' in url:
+                payload = '{"jsonrpc":"2.0","id":1,"result":"0x0000000000000000000000000000000000000000000000000000000000f1d3ed00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000000"}'  # noqa: E501
```

Comment:
that's probably fine an approach to have test-specific mocks. In the past we had these if/elsif/else mocks in utils etherscan test mocks.